src/auth/services/team.service.ts

```
@@ -130,75 +148,95 @@ export class TeamService {
-    if (!userMember.hasPermission("canEditJobs") && team.ownerId !== userId) {
-      throw new ForbiddenException("Insufficient permissions to update team")
+    const userMember = await this.getUserTeamMember(teamId, userId);
+    if (!userMember.hasPermission('canEditJobs') && team.ownerId !== userId) {
+      throw new ForbiddenException('Insufficient permissions to update team');
     }
 
     // Update team
-    Object.assign(team, updateTeamDto)
-    const updatedTeam = await this.teamRepository.save(team)
+    Object.assign(team, updateTeamDto);
+    const updatedTeam = await this.teamRepository.save(team);
```

ESLint

[error] 'updatedTeam' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)